docs/development/update_refdata.rst

```diff
@@ -10,50 +10,50 @@ We assume that you have added the necessary changes to TARDIS and have an open p
 
 #. Activate your ``tardis`` conda environment.
 
-    .. code-block:: None
```

Comment:
I looked through the sphinx documentation and I believe we want that to be `none` and not just blank (for all of them). Otherwise (I believe) this defaults to python highlighting, and this is not python code. The warning in the documentation build was because `none` is not supposed to be capitalized.

If you change that to `none` and the warnings return, then don't bother and I'll approve this, but if they don't return, then make that change please and then I will approve :)

Reply:
Hello @smithis7, I was trying to figure out that `none` keyword is a valid pygment lexer value or not ... found [this](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block) instead ... Here it states that the following for default value in the `code-block` directive:
![image](https://user-images.githubusercontent.com/66117751/119766934-8a252880-bed3-11eb-8e2b-3a7954274cfa.png)
Link [here](https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-highlight_language).

Reply:
@smithis7 Yes it works ... `none` is a valid `Pygment` Lexer Argument for `code-block` directive for `rst`. I will change it & push my changes :)